Pipfile

```diff
@@ -0,0 +1,21 @@
+[[source]]
```

Comment:
wouldn't it be better to put this in the allure directory.

this will makes it possible for us to submit a pullrequest to geocoder-tester adding `pienv` no ?